apps/src/netsim/NetSimLogBrowser.jsx

```diff
@@ -0,0 +1,159 @@
+            setRouterLogMode={this.props.setRouterLogMode}
+            localAddress={this.props.localAddress}
+            currentTrafficFilter={this.props.currentTrafficFilter}
+            setTrafficFilter={this.props.setTrafficFilter}
+          />
+          <div style={style.scrollArea}>
+            {/* TODO: get table sticky headers working */}
+            <NetSimLogBrowserTable
+              headerFields={this.props.headerFields}
+              logRows={this.props.logRows}
+              renderedRowLimit={this.props.renderedRowLimit}
+            />
+          </div>
+        </Body>
+      </Dialog>
+    );
+  }
+});
+export default NetSimLogBrowser;
+
+if (BUILD_STYLEGUIDE) {
+  const range = function (i) {
+    return new Array(i).fill().map((_, i) => i);
+  };
+
+  const randInt = function (i) {
+    return Math.floor(i * Math.random());
+  };
+
+  NetSimLogBrowser.styleGuideExamples = storybook => {
+    const i18n = {
+      logBrowserHeader_toggleMine: () => 'show my routers',
+      logBrowserHeader_toggleAll: () => 'show all routers',
+      logBrowserHeader_showAllTraffic: () => 'show all traffic',
+      logBrowserHeader_showMyTraffic: () => 'show my traffic',
+      logBrowserHeader_showTrafficFromMe: () => 'show traffic from me',
+      logBrowserHeader_showTrafficToMe: () => 'show traffic to me'
+    };
+
+    const simplePacket = [];
+    const fancyPacket = [
+      Packet.HeaderType.TO_ADDRESS,
+      Packet.HeaderType.FROM_ADDRESS,
+      Packet.HeaderType.PACKET_INDEX,
+      Packet.HeaderType.PACKET_COUNT
+    ];
+
+    const lipsumWords = `Lorem ipsum dolor sit amet,
+        consectetur adipiscing elit. Nam leo elit, tristique ac hendrerit vitae,
+        porta quis diam. Cras sit amet diam dapibus, ullamcorper nibh vitae,
+        luctus nisl. Curabitur fermentum accumsan commodo. Donec nec eros a
+        lorem tincidunt sodales ac sit amet odio. Duis eget ornare ante.`.split(/\s+/g);
+
+    const sampleData = new range(100).map(() => {
+      const routerNum = 1 + randInt(10);
+      const packetCount = 1 + randInt(4);
+      const packetNum = 1 + randInt(packetCount);
+      return {
+        'timestamp': Date.now() - randInt(600000),
+        'logged-by': `Router ${routerNum}`,
+        'status': Math.random() < 0.8 ? 'Success' : 'Dropped',
+        'from-address': `${routerNum}.${1 + randInt(13)}`,
+        'to-address': `${routerNum}.${1 + randInt(13)}`,
+        'packet-info': `${packetNum} of ${packetCount}`,
+        'message': range(randInt(20))
+            .map(() => lipsumWords[randInt(lipsumWords.length)])
+            .join(' ')
+      };
+    });
+
+    return storybook
+        .storiesOf('NetSimLogBrowser', module)
+        .addWithInfo(
+            'No filtering allowed',
+            `Here's what the dialog looks like with minimum settings.`,
+            () => (
+              <div id="netsim">
+                <div className="new-router-log-modal">
```

Comment:
are these required just for styling purposes? If so, how difficult would it be to move the styles into the React component?